site/en/guides/porting/implement-advanced-features.md

```diff
@@ -70,27 +70,32 @@ experience, we recommend enabling hardware acceleration instead of software to
 implement the above operations.
 
 To enable hardware acceleration within OpenThread, the following mbedTLS
-configuration header files should be added to the compile option flags in the
-platform example's Makefile:
+configuration header files should be added to the `ot-config` compile
+definitions in the platform's CMake files:
+
+[mbedtls-config.h]: https://github.com/openthread/openthread/blob/main/third_party/mbedtls/mbedtls-config.h
 
 -   Main configuration, which defines all necessary macros used in OpenThread:
-    `/openthread/third-party/mbedtls/mbedtls-config.h`
+    [`/openthread/third-party/mbedtls/mbedtls-config.h`][mbedtls-config.h]
```

Comment:
may assume in the platform repository and use the relative path here and other occurrences?

Reply:
@Vyrastas Will switching the link to a relative path mess with the `openthread.io` markdown import script?

Reply:
Unless it's referring to a file in this same repo (ot-docs), let's use the full path for now to avoid any issues with links.

Reply:
Ahh yeah. I forgot this is a different repo. Thanks!